server/dbutil/dbutil.go

```diff
@@ -0,0 +1,156 @@
+package dbutil
+
+import (
+	"crypto/md5"
+	"database/sql"
+	"fmt"
+	"log"
+	"os"
+
+	_ "github.com/mattn/go-sqlite3"
+	"github.com/pmezard/go-difflib/difflib"
+)
+
+const defaultExperimentID = 1
+
+// Bootstrap creates the necessary tables for the output DB. It is safe to call on a
+// DB that is already bootstrapped.
+func Bootstrap(db *sql.DB) error {
+	const (
```

Comment:
Any reason why not to extract all queries from everywhere to the top-level constants all together?

Reply:
Personal preference, I guess. I like to see the query contents near to where it is been used. But if the common style in other tools is to have all SQL queries as const at the top, that is also fine for me. Should I go ahead and move all queries?